File: URI_Python_UFRR/1012.py
linha1 = input().split(" ")

pi = 3.14159
A, B, C = linha1

a_ = (float(A) * float(C))/2
b_ = pi * (float(C)**2)
c_ = ((float(A) + float(B))*float(C))/2
d_ = float(B)**2
e_ = float(A) * float(B)

print('TRIANGULO: {:.3f}'.format(a_))
print('CIRCULO: {:.3f}'.format(b_))
print('TRAPEZIO: {:.3f}'.format(c_))
print('QUADRADO: {:.3f}'.format(d_))
print('RETANGULO: {:.3f}'.format(e_))

File: URI_C_UFRR/1012.c
#include <stdio.h>
int main(){

    double a,b,c,val;
    scanf("%lf %lf %lf",&a,&b,&c);
    val = (a * c)/2;
    printf("TRIANGULO: %.3lf\n", val);
    val = 3.14159 * (c*c);
    printf("CIRCULO: %.3lf\n", val);
    val = ((a + b)*c)/2;
    printf("TRAPEZIO: %.3lf\n", val);
    val = b*b;
    printf("QUADRADO: %.3lf\n", val);
    val = a*b;
    printf("RETANGULO: %.3lf\n", val);
    return 0;
}
